### PR TITLE
Add notes regarding ssl

### LINKED ISSUE #37: Problem: 1.8 upgrade instructions need to be edited
Need to edit the 1.8 upgrade instructions to include the option for AM without indexing

### LINKED ISSUE #44: Problem: document ssl usage
Configuring archivematica dashboard and storage-service using ssl, should be described here

### PR DESCRIPTION
**Pull request by scollazo**
fixes #44 
also fixes #37

### CODE REVIEW

**Comment by jhsimpson** [comment]
This fixed #44

**Comment by kellyannewithane** [comment]
@sevein or @sromkey can you CR?

**Comment by sromkey** [comment]
It looks good to me in terms of formatting but I don't really know if this is enough/the right information, so let's wait for @sevein's input.

File: admin-manual/installation/installation.rst [hunk 1]
```diff
@@ -1128,6 +1128,16 @@ It is possible to spread Archivematica's processing load across several machines
 
 For help, send an email to `Archivematica tech mailing list. <https://groups.google.com/forum/#!forum/archivematica-tech>`_
 
+.. _SSL-support:
+
+Using Archivematica through https
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Archivematica can be configured for https following the sample configs for `dashboard
```
Comment (sevein) [inline admin-manual/installation/installation.rst]: "configs" is not really a word, right? I think it should read "configurations".

File: admin-manual/installation/installation.rst [hunk 2]
```diff
@@ -1128,6 +1128,16 @@ It is possible to spread Archivematica's processing load across several machines
 
 For help, send an email to `Archivematica tech mailing list. <https://groups.google.com/forum/#!forum/archivematica-tech>`_
 
+.. _SSL-support:
+
+Using Archivematica through https
```
Comment (sevein) [inline admin-manual/installation/installation.rst]: I think I would write it differently, perhaps: "Configure Archivematica with SSL".

I've searched "using nginx through https": only one result. It doesn't seem very common.

File: admin-manual/installation/installation.rst [hunk 3]
```diff
@@ -1128,6 +1128,16 @@ It is possible to spread Archivematica's processing load across several machines
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Archivematica can be configured for https following the sample configs for `dashboard
+ <https://github.com/artefactual-labs/ansible-archivematica-src/blob/qa/1.7.x/templates/etc/nginx/sites-available/dashboard-ssl.conf.j2>`_ and `storage-service <https://github.com/artefactual-labs/ansible-archivematica-src/blob/qa/1.7.x/templates/etc/nginx/sites-available/storage-ssl.conf.j2>`_
```
Comment (sevein) [inline admin-manual/installation/installation.rst]: Missing period at the end of the sentence.

File: admin-manual/installation/installation.rst [hunk 4]
```diff
@@ -1128,6 +1128,16 @@ It is possible to spread Archivematica's processing load across several machines
...
+
+In order to obtain valid ssl certificates trusted by any browser, you can use `Let's Encrypt <https://letsencrypt.org>`_
```
Comment (sevein) [inline admin-manual/installation/installation.rst]: Missing period at the end of the sentence.

**Review by sevein — CHANGES_REQUESTED** [review]
A few suggestions. I also think that we should capitalize acronyms, e.g. **HTTPS**, **SSL**, etc...

**Review by sevein — APPROVED** [review]

**Pull request merged**